qiskit_experiments/database_service/utils.py

```diff
@@ -220,3 +220,25 @@ def copy(self):
         """Returns a copy of the list."""
         with self.lock:
             return self._container.copy()
+
+
+@dataclasses.dataclass(frozen=True)
```

Comment:
Do we need frozen?

Reply:
Not really but I meant

```python
def __init__(self, value, ...)
   self._value = value

@property
def value(self):
    return self._value
```

Reply:
Frozen seems fine, it stops you from accidentally changing something, and its not hard to make a new FitVal if you want to update a field.